src/Editor/Editing.cpp

```diff
@@ -542,6 +542,38 @@ struct EditingImpl : public Editing {
         gNotes->modify(edit, false, desc);
     }
 
+    void changeNoteSide() override {
+        Style* style = gStyle->get();
+
+        if (!style) {
+            HudNote("No style is currently active.");
+            return;
+        }
+
+        if (!style.id.endsWith("double")) {
```

Comment:
```suggestion
        if (!style->id.ends_with("double")) {
```
The string functions have changed to use std::string instead of Vortex::String